frontend/src/components/transactions/TransactionAction.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
-import * as T from "../../libraries/explorer-wamp/transactions";
+import { Row } from "react-bootstrap";
+import TransactionsApi, * as T from "../../libraries/explorer-wamp/transactions";
```

Comment:
https://github.com/near/near-explorer/blob/master/CONTRIBUTING.md#imports-ordering-and-grouping